### PR TITLE
NOX-301 Create or Update Owned Entity Translations

### PR DESCRIPTION
**Pull request by emir-sehovic1**

### CODE REVIEW

File: src/Nox.Docs/Templates/EntityEndpoints.template.md [hunk 1]
```diff
@@ -152,12 +152,19 @@ This section details the API endpoints related to enumeration attributes in a sp
...
+{{~ if entity.IsLocalized ~}}
 - **GET** `{{apiRoutePrefix}}/{{entity.PluralName}}/{key}/{{entity.PluralName}}Localized`
   - Description: Retrieve all {{entity.PluralName}}Localized for a specific {{entity.Name}}.
 
 - **PUT** `{{apiRoutePrefix}}/{{entity.PluralName}}/{key}/{{entity.PluralName}}Localized/{cultureCode}`
     - Description: Update or create values of {{entity.Name}}Localized for a specific {{entity.Name}}. Requires a payload with the new value of {{entity.Name}}LocalizedUpsertDto.
+{{~ end ~}}
+{{~ if entity.HasLocalizedOwnedRelationships ~}}
+{{ for localizedRelationship in ownedLocalizedRelationships }}
+- **PUT** `{{apiRoutePrefix}}/{{entity.PluralName}}/{key}/{{if localizedRelationship.IsWithMultiEntity}}{{localizedRelationship.OwnedEntity.PluralName}}{{else}}{{localizedRelationship.OwnedEntity.Name}}{{end}}Localized/{cultureCode}`
```
Comment (anna-naumova25) [inline src/Nox.Docs/Templates/EntityEndpoints.template.md]:
```suggestion
- **PUT** `{{apiRoutePrefix}}/{{entity.PluralName}}/{key}/{{GetNavigationPropertyName entity localizedRelationship}}Localized/{cultureCode}` 
```

File: src/Nox.Generator/Presentation/Api/OData/EntityController.Translations.template.cs [hunk 2]
```diff
@@ -83,4 +84,49 @@ public abstract partial class {{className}}Base
             
         return Ok(result);
     }
+    {{~ end ~}}
+    {{- for localizedRelationship in ownedLocalizedRelationships }}
+    [HttpPut("{{solution.Presentation.ApiConfiguration.ApiRoutePrefix}}/{{entity.PluralName}}/{{keysRoute}}{{if localizedRelationship.IsWithMultiEntity}}{{localizedRelationship.OwnedEntity.PluralName}}{{else}}{{localizedRelationship.OwnedEntity.Name}}{{end}}Localized/{%{{}%}{{cultureCode}}{%{}}%}")]
```
Comment (anna-naumova25) [inline src/Nox.Generator/Presentation/Api/OData/EntityController.Translations.template.cs]:
```suggestion
    [HttpPut("{{solution.Presentation.ApiConfiguration.ApiRoutePrefix}}/{{entity.PluralName}}/{{keysRoute}}{{GetNavigationPropertyName entity localizedRelationship}}Localized/{%{{}%}{{cultureCode}}{%{}}%}")]
```

**Review by anna-naumova25 — CHANGES_REQUESTED** [review]
Just several comments.

**Comment by emir-sehovic1** [comment]
> Just several comments.

Thanks @anna-naumova25 , I've added the changes

**Review by anna-naumova25 — DISMISSED** [review]

**Review by gjorgji-grgovski-authoritypartners — DISMISSED** [review]

**Review by gjorgji-grgovski-authoritypartners — DISMISSED** [review]

**Review by anna-naumova25 — APPROVED** [review]

**Review by gjorgji-grgovski-authoritypartners — APPROVED** [review]

**Pull request merged**